src/client/pydaos/raw/daos_api.py

```diff
@@ -1591,18 +1591,15 @@ def create(self, poh, con_prop=None, cb_func=None):
                                             self))
             thread.start()
 
-    def destroy(self, force=1, poh=None, cb_func=None):
+    def destroy(self, force=1, poh=None, uuid_str=None, cb_func=None):
```

Comment:
I agree. I got same feeling, so I removed uuid as a parameter of container.destroy() at the beginning. I reverted it back since 
 we do need to test c function daos_cont_destroy with non-existing uuid. Right. Calling container_destroy() for such tests may be a more reasonable way although it takes more effort.

Reply:
Yeah :( unfortunately too much work to handle on this PR :/

Reply:
> I agree. I got same feeling, so I removed uuid as a parameter of container.destroy() at the beginning. I reverted it back since we do need to test c function daos_cont_destroy with non-existing uuid. Right. Calling container_destroy() for such tests may be a more reasonable way although it takes more effort.

This comment explains a lot.  Lets move forward with this PR but I agree, once a object has poh, uuid set then it should use them and not take them as options for methods.

If you need to test closing non-existent uuid perhaps have the test update the object attribute rather than passing something into destroy?

Reply:
> If you need to test closing non-existent uuid perhaps have the test update the object attribute rather than passing something into destroy?

Yes. I thought about replacing the uuid attribute of the container object for testing non-existent uuid. We need to save current uuid, then update it with a random uuid then call destroy(). When destroy() fails, we need to restore uuid attribute then try again destroy(). Do you prefer this way in future?